projects/demo/src/app/app.component.html

```diff
@@ -1,106 +1,74 @@
-        <ng-template valError="pattern">The last name must only contain letters</ng-template>
-      </val-errors>
-    </div>
+<main role="main" class="container mt-5">
 
-    <div class="form-group">
-      <label for="age">Age (required, min 18 -> custom message with error exposed)</label>
-      <input id="age" formControlName="age" type="number" class="form-control"/>
-      <val-errors controlName="age" label="The age">
-        <ng-template valError="min" let-error="error">You must be at least {{ error.min | number }} years old</ng-template>
-      </val-errors>
+  <div class="jumbotron d-sm-flex">
+    <div class="logo">
+      <img src="/assets/logo.svg" alt="logo" />
     </div>
-
-    <div formGroupName="credentials">
-      <div class="form-group">
-        <label for="password">Password (required)</label>
-        <input id="password" formControlName="password" class="form-control" type="password"/>
-        <val-errors controlName="password" label="The password"></val-errors>
-      </div>
-
-      <div class="form-group">
-        <label for="password">Password confirmation (required)</label>
-        <input id="confirmation" formControlName="confirmation" class="form-control" type="password"/>
-        <val-errors controlName="confirmation" label="The password confirmation"></val-errors>
-        <val-errors [control]="form.get('credentials')">
-          <ng-template valError="validCredentials">The confirmation <strong>must</strong> match the password</ng-template>
-        </val-errors>
-      </div>
+    <div class="flex-fill">
+      <h2>ngx&#8209;valdemort <small>by&nbsp;Ninja&nbsp;Squad</small></h2>
+      <p class="lead">
+        Simple, consistent validation error messages for your Angular forms.
+      </p>
     </div>
+  </div>
 
-    <button class="btn btn-primary">Submit</button>
+  <h2>The problem</h2>
 
-  </form>
+  <demo-problem></demo-problem>
 
-  <hr/>
+  <h2>Our solution: ngx-valdemort</h2>
 
-  <form ngSubmit="submitPerson()">
-    <div class="form-group">
-      <label for="name">Name (required)</label>
-      <input id="name" class="form-control" name="name" [(ngModel)]="person.name" required #n="ngModel"/>
-      <val-errors controlName="name" label="The name"></val-errors>
-    </div>
+  <demo-solution></demo-solution>
 
-    <div ngModelGroup="address">
-      <div class="form-group">
-        <label for="street">Street (required)</label>
-        <input id="street" class="form-control" name="street" [(ngModel)]="person.address.street" required />
-        <val-errors controlName="street" label="The street"></val-errors>
-      </div>
-
-      <div class="form-group">
-        <label for="city">City (required)</label>
-        <input id="city" class="form-control" name="city" [(ngModel)]="person.address.city" required #cityCtrl="ngModel" />
-        <val-errors [control]="cityCtrl.control" label="The city"></val-errors>
-      </div>
-    </div>
+  <h2>Consistent error messages</h2>
```

Comment:
"Want to use consistent error messages across all forms? We've got you covered!"
Too long, but something like this?

Reply:
How about 

> Consistent error messages? We've got you covered!

Reply:
Yup nicer. I can't really come with a better alternative, but I have the feeling that "Consistent" is too vague to describe that you have the same message errors across all forms by default. But it's good enough.